src/controllers/dynakube/activegate/internal/tenantinfo/reconciler_test.go

```diff
@@ -0,0 +1,97 @@
+package tenantinfo
+
+import (
+	"context"
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/dtclient"
+	"github.com/Dynatrace/dynatrace-operator/src/scheme"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+const (
+	testDynakubeName = "test-dynakube"
+	testNamespace    = "test-namespace"
+)
+
+var (
+	tenantInfoResponse = &dtclient.ActiveGateTenantInfo{
+		TenantInfo: dtclient.TenantInfo{
+			UUID:  "testUUID",
+			Token: "dt.some.valuegoeshere",
+		},
+		Endpoints: "someEndpoints",
+	}
+)
+
+func newTestReconciler(client client.Client) *Reconciler {
```

Comment:
`newTestReconciler` is not descriptive enought since it also creates a new instance, not only a reconciler.
Maybe something like `newTestReconcilerWithInstance` or `newTestEnvironment` would be more fitting
I personally would prefer the former as `newTestEnvironment` means different things in different contexts

Reply:
This is also true for the other constructor